state/postgres_relation_stats.go

```diff
@@ -38,6 +38,7 @@ type PostgresRelationStats struct {
 	Reltuples         float32   // Number of live rows in the table. -1 indicating that the row count is unknown
 	Relallvisible     int32     // Number of pages that are marked all-visible in the table's visibility map
 	ExclusivelyLocked bool      // Whether these statistics are zeroed out because the table was locked at collection time
+	ToastReltuples    int32     // Number of live rows in the TOAST table. -1 indicating that the row count is unknown
```

Comment:
I copied the -1 comment from the other reltuples field. Is this accurate here? I think it may be zero if the table doesn't use any TOAST.

Note: be sure to change the protobuf comment as well.

Reply:
Update I changed the `coalesce` call to default to -1 so now internally, the field will be null for old collector versions, and -1 for tables missing TOAST (or where the row count is unknown).